storage/persistent_storage/persistent-storage-ocs.adoc

```diff
@@ -58,7 +58,7 @@ OpenShift Container Storage on top of Red Hat Hyperconverged Infrastructure (RHH
 |link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.7/html/deploying_and_managing_openshift_container_storage_using_red_hat_openstack_platform/[Deploying and managing OpenShift Container Storage 4.7 using Red Hat OpenStack Platform]
 
 |Deploying and managing Red Hat OpenShift Container Storage on {rh-virtualization-first}
-|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.7/html/deploying_and_managing_openshift_container_storage_using_red_hat_virtualization_platform/[Deploying and managing OpenShift Container Storage 4.7 using Red Hat Virtualization Platform]
+|link:!https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.7/html/deploying_and_managing_openshift_container_storage_using_red_hat_virtualization_platform/[Deploying and managing OpenShift Container Storage 4.7 using Red Hat Virtualization Platform]
```

Comment:
fyi @bobfuru

Reply:
Looks like it's now this:
https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.7/html/deploying_openshift_container_storage_using_red_hat_virtualization_platform/index